admin/environment-management/cvms.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Docker in Environments
+description: Learn how to enable support for secure Docker inside Environments. 
+---
+
+As a platform administrator, you have the option to enable
+[Container-based Virtual Machines](../../environments/cvms.md) as an Environment
+deployment option. This option allows users to run system-level
+programs inside their Environment including Docker and systemd.
+
+Navigate to **Manage > Admin > General** to enable this option.
+
+## Infrastructure requirements
+
+- The Kubernetes Nodes must have a minimum kernel version of `5.4`
+(released Nov 24th, 2019).
+- The Kubernetes Nodes must be running an Ubunutu OS.
+- Legacy versions of cluster-wide proxy services such as Istio are not
```

Comment:
Do you know which versions specifically?

Reply:
Istio <= 1.14 is supported. @deansheather has better intel on these verisons.

Reply:
The phrasing here is intentionally vague until we have additional confidence for the combinations of Istio version and Kube version that work properly. Basically, not exactly

Reply:
Istio 1.5 or lower

Or newest Istio and k8s 1.19

Reply:
Actually I meant 1.4 or lower. My bad

Reply:
Actually it's 1.4 or lower

Reply:
Let's keep it vague until we're more confident from our own testing or until we have evidence from more customers that it's compatible with their cluster configurations.